isis/config/exclude_list.txt

```diff
@@ -0,0 +1,13 @@
+- /a_older_versions/*
+- /kernels/*/former_versions/
+- /corrupt_files/*
+- /zzarchive/**
+- /kernels/*/original
+- gallileo/kernels/ck/prime_mission/*/extended_mission/*
+- gallileo/kernels/*/prime_mission/
+- galileo/kernels/ck/GEM/
+- clementine1/kernels/ck/save
+- chandrayaan1/kernels/spk/SAVE_SCS_2017-11-22
+- */kernels/fk/release.*/*
+- cassini/kernels/fk/Archive
```

Comment:
Users need to download the script, to save them from having to downloading another file, we should probably put this as a list inside the script.